src/search/index.js

```diff
@@ -423,6 +423,15 @@ export default class CoProcureSearch extends HTMLElement {
     document.head.appendChild(script2);
   }
 
+  prepareSearchQueryTerm(term) {
+    // AWS CloudSearch chokes on single quotes in queries unless they're escaped.
+    // Escaping backslashes must be added *after* URI encoding, otherwise they'll be cleaned out.
+    let preparedTerm = encodeURIComponent(term)
+    preparedTerm = preparedTerm.replace(/'/g, "%5C'");
```

Comment:
Does encodeURIComponent take care of the backslashes?

Reply:
`encodeURIComponent` will convert existing backslashes to %5C, but in this case I'm trying to add *new* backslashes in front of all apostrophes. Another approach would be to do this above the `encodeURIComponent` call, but I think I was having trouble getting the backslashes into the URLs (since they're also escape characters), so I ultimately found it simpler to just prepend the already-converted `%5C`s instead.

Does that seem alright?